wordpress-mobile/WordPress-iOS
# Fix app freeze on fresh install when access the dashboard

Fixes #18256 

## Testing Instructions

1. Uninstall the WordPress app from the simulator. ⚠️ NOTE: This issue only happens on a fresh install. ⚠️
2. Build the app using Xcode 13.3, targeting iOS 15.4.
3. Log in
4. Select a site that doesn't have draft or scheduled posts
5. Tap on the Home section.
6. Observe that the app does not freeze.

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

5. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
